main.py

```diff
@@ -1,24 +1,38 @@
 import os
 import shutil
+import re
 
-# Taking directory path as an input from user.
-directory = input("Enter the directory path") 
-
+directory = input("Enter the directory path :")
+sub_dir =[]
 images = []
 count = 0
 
+
+for item in os.listdir(directory):
+    if os.path.isdir(os.path.join(directory,item)):
+        sub_dir.append(item)
+
+pattern = r'Image_(\d)'
+for folder in sub_dir:
+    match = re.match(pattern, folder)
+    if match:
+        series = int(match.group(1))
+        if series > count:
+            count = series
```

Comment:
Have the method to find the latest count of the folders to continue from contained within its own method or function